grammar.js

```diff
@@ -234,6 +235,7 @@ module.exports = grammar({
     keyword_nowait: _ => make_keyword("nowait"),
     keyword_attribute: _ => make_keyword("attribute"),
     keyword_authorization: _ => make_keyword("authorization"),
+    keyword_action: _ => make_keyword("action"),
```

Comment:
this new keyword also needs to be added in the highlights.scm

Reply:
maybe as a `@type.qualifier` ?

Reply:
I put it into keyword.